contrib/test_mad.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+set -exE -o pipefail
+
+cd "$BUILD_SOURCESDIRECTORY"
+
+run_mad_server() {
+	build_ucx
```

Comment:
indent

Reply:
Unified indent by auto-formating.
While seemingly the same, some indents used Tabs, while others 4 spaces.